elasticsearch/linux_distributions/opendistro-onetime-setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# This script performs one-time setup for the Open Distro for ElasticSearch tarball distribution.
+# It installs a demo security config and sets up the performance analyzer
+
+ES_HOME=`dirname $(realpath $0)`; cd $ES_HOME
```

Comment:
@sreekarjami Isnt the `realpath` is changed for macos? Or you have another script for macos specifically in macos directory?

Reply:
Yes, I have separate script for macos which removed realpath.

Reply:
I don't want to mix functional changes with the structural change in this PR. We can address MacOS issues in a new commit.